src/core/lib/iomgr/ev_epoll_linux.c

```diff
@@ -180,8 +217,10 @@ static void fd_unref(grpc_fd *fd);
 static void fd_global_init(void);
 static void fd_global_shutdown(void);
 
-#define CLOSURE_NOT_READY ((grpc_closure *)0)
-#define CLOSURE_READY ((grpc_closure *)1)
+#define CLOSURE_NOT_READY ((gpr_atm)0)
+#define CLOSURE_READY ((gpr_atm)1)
```

Comment:
This needs to be !=1, else there's no way to distinguish a shutdown with GRPC_ERROR_NONE... suggest 2